### PR TITLE
jsonnet/kube-prometheus: update Prometheus dependency to v2.22.0

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -657,7 +657,7 @@ In case you want to monitor all namespaces in a cluster, you can add the followi
 [embedmd]:# (examples/all-namespaces.jsonnet)
 ```jsonnet
 local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
-  (import 'kube-prometheus/kube-prometheus-all-namespaces.libsonnet') + {
+           (import 'kube-prometheus/kube-prometheus-all-namespaces.libsonnet') + {
```
Comment (s-urbaniak) [inline README.md:660]: out of curiosity: is this a result of `jsonnetfmt`ing?

**Comment by s-urbaniak** [comment]
just one question, else lgtm :+1:

File: README.md (hunk 1 above)
Comment (simonpasquier) [reply README.md:660]: IIUC the CI only `make generate`, not `make generate fmt`. Maybe it should? cc @paulfantom

Comment (paulfantom) [reply README.md:660]: Yes, this is a result of `jsonnetfmt`.

@simonpasquier Maybe you run `make` at some point? This includes `fmt` target :shrug:

**Review by paulfantom — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (simonpasquier) [reply README.md:660]: yep I think I executed `make`

**Pull request merged**